File: src/main/kotlin/de/check24/learningKotlinJava/java/If.java
package de.check24.learningKotlinJava.java;

public class If {
    public static void main(String[] args) {
        int zahl = 2;
        if (zahl <= 1){
            System.out.println(zahl + " kleiner oder gleich 1");
        } else if (zahl == 2) {
            System.out.println(zahl + " ist gleich 2");
        } else {
            System.out.println(zahl + " ist größer als 2");
        }
    }
}

File: src/main/kotlin/de/check24/learningKotlinJava/kotlin/If.kt
package de.check24.learningKotlinJava.kotlin

fun main() {
    var zahl: Int = 2
    if (zahl <= 1) {
        println("$zahl kleiner oder gleich 1")
    } else if (zahl == 2) {
        println("$zahl ist gleich 2")
    } else {
        println("$zahl ist größer als 2")
    }
}